etc/cvmfs/config.d/ligo.osgstorage.org.conf

```diff
@@ -1,5 +1,6 @@
 # stashcache servers configured by OSG
-CVMFS_EXTERNAL_URL="https://xrootd-local.unl.edu:1094//user/ligo/;https://stashcache.t2.ucsd.edu:8443//user/ligo/;https://its-condor-xrootd1.syr.edu:8443//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8443//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8443//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8443//user/ligo/;https://osg-gftp2.pace.gatech.edu:8443//user/ligo/;https://dtn2-daejeon.kreonet.net:8443//user/ligo/;https://osg-houston-stashcache.nrp.internet2.edu:8443//user/ligo/;https://osg-sunnyvale-stashcache.nrp.internet2.edu:8443//user/ligo/;https://ligo.hpc.swin.edu.au:8443//user/ligo/"
+CVMFS_EXTERNAL_URL="https://xrootd-local.unl.edu:1094//user/ligo/;https://stashcache.t2.ucsd.edu:8443//user/ligo/;https://its-condor-xrootd1.syr.edu:8443//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8443//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8443//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8443//user/ligo/;https://osg-gftp2.pace.gatech.edu:8443//user/ligo/;https://dtn2-daejeon.kreonet.net:8443//user/ligo/;https://osg-houston-stashcache.nrp.internet2.edu:8443//user/ligo/;https://osg-
```

Comment:
Something's missing here. "sunnyvale" is replaced by a newline and "vale".

Reply:
fixed